### PR TITLE
[New] Admin Dashboard

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**
- Created icons to show participation and growth data.
- Using library "bizcharts" to create and render the three "top five" bar charts and the line chart.

### CODE REVIEW

File: services/frontend-v3/src/pages/admin/Competency.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// import React, { Component } from "react";
+// import AdminOptionEdit from "../components/admin/AdminOptionEdit";
```
Comment (dynamic11) [inline services/frontend-v3/src/pages/admin/Competency.jsx]: why is it all commented out? should we just delete it?

File: services/frontend-v3/src/pages/admin/Diploma.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// import React, { Component } from "react";
+// import AdminOptionEdit from "../components/admin/AdminOptionEdit";
```
Comment (dynamic11) [inline services/frontend-v3/src/pages/admin/Diploma.jsx]: should we just delete this file?

File: services/frontend-v3/src/pages/admin/Diploma.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (dynamic11) [inline services/frontend-v3/src/pages/admin/Diploma.jsx]: delete?

File: services/frontend-v3/src/pages/admin/Diploma.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// class AdminDiploma extends Component {
```
Comment (dynamic11) [inline services/frontend-v3/src/pages/admin/Diploma.jsx]: delete?

**Review by dynamic11 — DISMISSED** [review]
Please delete all commented out files. You shouldn't commit these file

File: services/frontend-v3/src/pages/admin/Competency.jsx (hunk 1 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/pages/admin/Competency.jsx]: Deleted

File: services/frontend-v3/src/pages/admin/Diploma.jsx (hunk 2 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/pages/admin/Diploma.jsx]: Deleted

File: services/frontend-v3/src/pages/admin/Diploma.jsx (hunk 3 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/pages/admin/Diploma.jsx]: Yes

File: services/frontend-v3/src/pages/admin/Diploma.jsx (hunk 4 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/pages/admin/Diploma.jsx]: Deleted

**Review by Rizvida1 — APPROVED** [review]
Make sure that the code compiles without warnings for your files.

**Pull request merged**